docs/10-clusters.md

```diff
@@ -27,6 +27,32 @@ NOTE: The cluster id (--id) can only contain alphanumeric characters including h
 #### Low priority nodes
 You can create your cluster with [low-priority](https://docs.microsoft.com/en-us/azure/batch/batch-low-pri-vms) VMs at an 80% discount by using **--size-low-pri** instead of **--size**. Note that these are great for experimental use, but can be taken away at any time. We recommend against this option when doing long running jobs or for critical workloads.
 
+#### Configuration
```

Comment:
i dont 100% that these docs should go here. I think we should make a **docs/11-cluster-config.md**

The cluster config stuff is for a slightly different use case. Also i think that this cluster config section will be large enough eventually (with all the other config work) that it will be warrant having its own page just for space.